packages/measure-tools/README.md

```diff
@@ -0,0 +1,162 @@
+# @bentley/measure-tools-react
+
+Copyright © Bentley Systems, Incorporated. All rights reserved.
+
+The measure-tools-react package provides measurement tools and React components to access within an iTwin Viewer.
+
+Every iModel.js application has initialization code where they must call ```IModelApp.startup()```. This library has a similar initialization entrypoint that must be called to configure localization and register tools.
+Call the  ```MeasureTools.startup() ``` **AFTER** the IModelApp one.
+
+```typescript
+    await IModelApp.startup(opts);
+    await MeasureTools.startup();
+```
+
+After that, most of your code changes will be setting up your frontstage to consume the default measurement tools (such as measure distance, polygon, location, etc).
+
+**Note:** Startup can be called any number of times but will only startup once. Some other libraries that use measure-tools may have already initialized it for you!
+
+# What to add in your iTwin-Viewer based app (UI 2.0)
```

Comment:
Can you refer to it as "iTwin App UI" rather than "UI 2.0"?  Trying to remove the very useless name of "UI 2.0" from all docs :)

Reply:
Should i remove UI 1.0 thats mentioned below?